screens/UsingScreen.js

```diff
@@ -16,49 +16,60 @@ const fredVictorCoordinates = {
 // TODO: Remember to change this to 500 prior to Beta testing
 const MAXIMUM_DISTANCE = 1000000000 // meters
 
-const UsingScreen = (props) => {
 
-  const { time, userId, token } = props;
-  const { timeRemaining } = time
+class UsingScreen extends Component {
+  
+  constructor(props) {
+    super(props);
+    this.startAlarm.bind(this);
+  }
 
-  const startAlarm = () => {
-    const distance = computeDistance(fredVictorCoordinates, props.location.coords)
-    if( distance > MAXIMUM_DISTANCE ) Alert.alert("Cannot start alarm", "There are no responders within your area", [
+  startAlarm = () => {
+    const { time, userId, token } = this.props;
+    const { timeRemaining } = time
+    const distance = computeDistance(fredVictorCoordinates, this.props.location.coords)
+    if (distance > MAXIMUM_DISTANCE) Alert.alert("Cannot start alarm", "There are no responders within your area", [
       { text: 'OK' }
     ], { cancelable: false })
     else {
       Actions.alarm();
 
-      props.makeAlarmLog(userId, timeRemaining, token);
+      this.props.makeAlarmLog(userId, timeRemaining, token);
     }
   }
 
-  return (
-    <Container>
-    <Header leftButton="menu" onLeftButtonPress={() => Actions.drawerOpen()}>Using Mode</Header>
+  componentDidMount(){
+    this.props.getNumberOfAvailableResponders(this.props.userId, this.props.token);
+  }
```

Comment:
Instead of turning into react component can use `useEffect`

```suggestion
  useEffect(() => {
    props.getNumberOfAvailableResponders(props.userId, props.token);
  }, [])
```

but I think it's ok if it's done!

Reply:
This can be a separate task. I have seen a lot of differences in class and functional components and feel like we should stick with either one. I personally favor functional components as it tends to steer towards less code. Same issue with state management; I dont know if it is a common convention to just take care of everything with redux even when a state variable is only used on one component. What do y'all think?